packages/orbit-components/src/ChoiceGroup/components/FilterWrapper.tsx

```diff
@@ -16,67 +13,28 @@ interface Props {
   ) => void | Promise<void>;
 }
 
-const hoverAndFocus = () => css`
-  background-color: ${({ theme }) => theme.orbit.paletteBlueLight};
-
-  ${StyledOnlyButton} {
-    visibility: visible;
-    opacity: 1;
-  }
-`;
-
-const StyledContentWrapper = styled.div<{ disabled: boolean }>`
-  ${({ disabled }) => css`
-    box-sizing: border-box;
-    width: 100%;
-    padding-left: 4px;
-    border-radius: 4px;
-    display: flex;
-    align-items: center;
-    height: ${({ theme }) => theme.orbit.heightButtonSmall};
-
-    ${StyledOnlyButton} {
-      visibility: hidden;
-      opacity: 0;
-    }
-
-    ${!disabled &&
-    css`
-      @media (hover: none) {
-        ${StyledOnlyButton} {
-          visibility: visible;
-          opacity: 0.3;
-          &:hover {
-            opacity: 1;
-          }
-        }
-      }
-
-      @media (hover) and (pointer: fine) {
-        &:hover {
-          ${hoverAndFocus};
-        }
-
-        &:focus-within {
-          ${hoverAndFocus}
-        }
-      }
-    `};
-  `}
-`;
-
-StyledContentWrapper.defaultProps = {
-  theme: defaultTheme,
-};
-
 const FilterWrapper = ({ child, children, onOnlySelection, onlySelectionText }: Props) => {
   const { value, label, disabled } = child.props;
 
   return (
-    <StyledContentWrapper disabled={disabled}>
+    <div
+      className={cx(
+        "h-form-box-small box-border flex w-full items-center rounded-[4px] pl-[4px]",
```

Comment:
isn't there a `rounded-sm` or `rounded-xs` instead of `rounded-[4px]`? same for `pl-[4px]`